2025/ag-wg.html

```diff
@@ -150,111 +149,143 @@ <h1 id="title">DRAFT Accessibility Guidelines Working Group Charter</h1>
 
-        <p>The Accessibility Guidelines Working Group describes the characteristics of accessible web content, in the form of guidance to content developers. It also describes support from other components needed for its advice to work, such as technologies, user agents, and authoring tools. Guidelines from this Working Group are widely used by developers and implementers, referenced in industry policy, and benefit all web users.</p>
+        <p>The Accessibility Guidelines Working Group describes the characteristics of accessible web content, in the form of guidance to content creators. It also describes support from other components needed for its advice to work, such as technologies, user agents, and authoring tools. Guidelines from this Working Group are widely used by developers and implementers, referenced in industry policy, and benefit all web users.</p>
       </section>
 
-      <section id="background-focus">
-        <h3>Focus</h3>
-        <p><span class="todo">Is this needed for this charter, or would Scope be sufficient?</span></p>
-      </section>
-      
       <section id="scope" class="scope">
         <h2>Scope</h2>
-        <p><i class="todo">What exactly is in scope, and out of scope.
-          For legal review. Be brief.</i></p>
 
-        <section id="section-out-of-scope">
-          <h3 id="out-of-scope">Out of Scope</h3>
-            <p>The following features are out of scope, and will not be addressed by this Working group.</p>
+        <section id="wcag3">
+          <h3>WCAG 3</h3>
+
+          <p>The primary focus of this charter will be to publish a Candidate Recommendation for the <a href="https://www.w3.org/TR/wcag-3.0/">W3C Accessibility Guidelines (WCAG) 3</a>. This will include a period of wide review. This publication will include:</p>
+
+          <ul>
+            <li>A full set of guidelines and requirements,</li>
+            <li>A conformance model,</li>
+            <li>Comprehensive informative user needs for each guideline,</li>
+            <li>At least one informative test for each requirement, and</li>
+            <li>At least one method for each requirement.</li>
+          </ul>
+        </section>
+
+        <p>WCAG benefits hugely from extensive informative materials that help foster understanding of the requirements, why they are important and how to test them. This will be developed in line with the core recommendation but is not necessary its publication.</p>
```

Comment:
```suggestion
        <p>WCAG benefits hugely from extensive informative materials that help foster understanding of the requirements, why they are important and how to test them. This will be developed in line with the core recommendation but is not necessary before its publication.</p>
```